cibuildwheel/windows.py

```diff
@@ -8,89 +8,72 @@
-        return None
     python_name = "python" if configuration.version[0] == '3' else "python2"
     if configuration.arch == "32":
         python_name = python_name + "x86"
     return [python_name, "-Version", configuration.nuget_version, "-OutputDirectory", "C:/python"]
 
 def get_python_configurations(build_selector):
-    PythonConfiguration = namedtuple('PythonConfiguration', ['version', 'arch', 'identifier', 'path', "nuget_version"])
+    PythonConfiguration = namedtuple('PythonConfiguration', ['version', 'arch', 'identifier', 'nuget_version'])
     python_configurations = [
-        PythonConfiguration(version='2.7.x', arch="32", identifier='cp27-win32', path='C:\\Python27', nuget_version="2.7.16"),
-        PythonConfiguration(version='2.7.x', arch="64", identifier='cp27-win_amd64', path='C:\\Python27-x64', nuget_version="2.7.16"),
-        PythonConfiguration(version='3.4.x', arch="32", identifier='cp34-win32', path='C:\\Python34', nuget_version=None),
-        PythonConfiguration(version='3.4.x', arch="64", identifier='cp34-win_amd64', path='C:\\Python34-x64', nuget_version=None),
-        PythonConfiguration(version='3.5.x', arch="32", identifier='cp35-win32', path='C:\\Python35', nuget_version="3.5.4"),
-        PythonConfiguration(version='3.5.x', arch="64", identifier='cp35-win_amd64', path='C:\\Python35-x64', nuget_version="3.5.4"),
-        PythonConfiguration(version='3.6.x', arch="32", identifier='cp36-win32', path='C:\\Python36', nuget_version="3.6.8"),
-        PythonConfiguration(version='3.6.x', arch="64", identifier='cp36-win_amd64', path='C:\\Python36-x64', nuget_version="3.6.8"),
-        PythonConfiguration(version='3.7.x', arch="32", identifier='cp37-win32', path='C:\\Python37', nuget_version="3.7.4"),
-        PythonConfiguration(version='3.7.x', arch="64", identifier='cp37-win_amd64', path='C:\\Python37-x64', nuget_version="3.7.4")
+        PythonConfiguration(version='2.7.x', arch="32", identifier='cp27-win32', nuget_version="2.7.17"),
+        PythonConfiguration(version='2.7.x', arch="64", identifier='cp27-win_amd64', nuget_version="2.7.17"),
+        PythonConfiguration(version='3.5.x', arch="32", identifier='cp35-win32', nuget_version="3.5.4"),
+        PythonConfiguration(version='3.5.x', arch="64", identifier='cp35-win_amd64', nuget_version="3.5.4"),
+        PythonConfiguration(version='3.6.x', arch="32", identifier='cp36-win32', nuget_version="3.6.8"),
+        PythonConfiguration(version='3.6.x', arch="64", identifier='cp36-win_amd64', nuget_version="3.6.8"),
+        PythonConfiguration(version='3.7.x', arch="32", identifier='cp37-win32', nuget_version="3.7.5"),
+        PythonConfiguration(version='3.7.x', arch="64", identifier='cp37-win_amd64', nuget_version="3.7.5"),
+        PythonConfiguration(version='3.8.x', arch="32", identifier='cp38-win32', nuget_version="3.8.0"),
+        PythonConfiguration(version='3.8.x', arch="64", identifier='cp38-win_amd64', nuget_version="3.8.0"),
```

Comment:
We could potentially even make `version` be the `nuget_version`, or not?
Or keep it like in `macOS.py` and remove the `.x`; any reason to have it here?

Reply:
The version with `.x` is used in the code to set an environment variable. It's for the Appveyor build tool selection but reading the script, `x` could be anything. So yes, we could use the same field. I'll do that.